build/generate-manifests.sh

```diff
@@ -45,6 +45,10 @@ function checkDependencies() {
 
 function generateCRDs() {
   go run -mod=vendor github.com/operator-framework/operator-sdk/cmd/operator-sdk generate crds --crd-version v1
+
+  # workaround for https://github.com/kubernetes-sigs/controller-tools/issues/457
```

Comment:
Should we create an issue to track this?

Reply:
Good point. I filed https://issues.redhat.com/browse/MAISTRA-1721